PCP_Chapter4_Exercises.cpp

```diff
@@ -14,51 +14,69 @@
 // Questions:
 //  How can I call a class that is in a different source file to the main function? 
 
-#include "stdafx.h"
+// use "using namespace blabla" sparingly - in general it is better to type out
+// the whole namespace to avoid ambiguity, and make the source clear.
+
+#include <iostream>
 using namespace std;
 
-// Called ex_42 for the exercise problem numbered 4.2 in the book. Holds functions that return a letter block (So far Hello) 
-class ex_42 {
-
-	public:
-		string Letter_H()
-		{ return "*   *\n*   *\n*   *\n*****\n*   *\n*   *\n*   *\n"; }
-		string Letter_E()
-		{ return "*****\n*\n*\n*****\n*\n*\n*****\n"; }
-		string Letter_L()
-		{ return "*\n*\n*\n*\n*\n*\n*****\n"; }
-		string Letter_O()
-		{ return " *** \n*   *\n*   *\n*   *\n*   *\n*   *\n *** \n\n"; }
-	
+// Called Ex42 for the exercise problem numbered 4.2 in the book. Holds functions that return a letter block (So far Hello)
+class Ex42 {
+
+    public:
+        string Letter_H()
+        {
+            return "*   *\n*   *\n*   *\n*****\n*   *\n*   *\n*   *\n";
+        }
+
+        string Letter_E()
+        {
+            return "*****\n*\n*\n*****\n*\n*\n*****\n";
+        }
+
+        string Letter_L()
+        {
+            return "*\n*\n*\n*\n*\n*\n*****\n";
+        }
+
+        string Letter_O()
+        {
+            return " *** \n*   *\n*   *\n*   *\n*   *\n*   *\n *** \n\n";
+        }
+
 };
 
 // Class for exercise 4.3 in the book. Functions calculate the area and circumference of a rectangle. 
-class ex_43 {
+class Ex43 {
 
 public:
-	int rec_area(int a_height, int a_width) // first parameter holds the height of the rectangle, second holds width and returns area.
-	{ return a_height * a_width; }
-
-	int rec_circum(int c_height, int c_width) // first parameter holds the height of the rectangle, second holds width  and returns circumference.
-	{ return 2 * (c_height + c_width); }
-	
-	//If i were to chose the values 6.8 and 2.3 I'd probably change the function type to "float" since these are decimal numbers. 
-	
+    int rec_area(int a_height, int a_width) // first parameter holds the height of the rectangle, second holds width and returns area.
+    {
+        return a_height * a_width;
+    }
+
+    int rec_circum(int c_height, int c_width) // first parameter holds the height of the rectangle, second holds width  and returns circumference.
+    {
+        return 2 * (c_height + c_width);
+    }
+
+    //If i were to chose the values 6.8 and 2.3 I'd probably change the function type to "float" since these are decimal numbers. 
+
 };
 
 
-int _tmain(int argc, _TCHAR* argv[])
```

Comment:
This function name is Microsoft specific, and it is ok to use if you know that this code will never run in non-Windows environment. In general, try to write code independent of the platform - so if the need ever arises to port it from Windows to Linux there is less work to do.